### ISSUE TITLE
Rails/BulkChangeTable error on empty migration

### ISSUE THREAD

**Issue by andyw8** [rubocop-hq/rubocop#5958]
Rails/BulkChangeTable fails on a migration file with no entries.

## Expected behavior

No warnings should be reported.

## Actual behavior

Fails with:

```
An error occurred while Rails/BulkChangeTable cop was inspecting ....../db/migrate/20180526194248_drop_delayed_jobs.rb:2:2.
undefined method `each_child_node' for nil:NilClass
.../lib/rubocop/cop/rails/bulk_change_table.rb:141:in `on_def'
.../lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
.../lib/rubocop/cop/commissioner.rb:105:in `with_cop_error_handling'
.../lib/rubocop/cop/commissioner.rb:56:in `block in trigger_responding_cops'
.../lib/rubocop/cop/commissioner.rb:55:in `each'
.../lib/rubocop/cop/commissioner.rb:55:in `trigger_responding_cops'
.../lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
.../lib/rubocop/ast/traversal.rb:88:in `on_class'
.../lib/rubocop/cop/commissioner.rb:34:in `block (2 levels) in <class:Commissioner>'
.../lib/rubocop/ast/traversal.rb:12:in `walk'
.../lib/rubocop/cop/commissioner.rb:45:in `investigate'
.../lib/rubocop/cop/team.rb:114:in `investigate'
.../lib/rubocop/cop/team.rb:102:in `offenses'
.../lib/rubocop/cop/team.rb:44:in `inspect_file'
```

## Steps to reproduce the problem

Run RuboCop on the following:

```
class DropDelayedJobs < ActiveRecord::Migration[5.1]
  def change; end
end
```

## RuboCop version

0.57.0 (using Parser 2.5.0.5, running on ruby 2.5.0 x86_64-darwin16)
